### PR TITLE
[POL:CoreTeam.vue] Polished Desktop view of /team

### PR DESCRIPTION
**Pull request by ShantanuLamba**

### CODE REVIEW

File: frontend/src/pages/toplevel/CoreTeam.vue [hunk 1]
```diff
@@ -107,4 +107,4 @@ export default {
         row-gap: 2rem;
```
Comment (aman2712) [inline frontend/src/pages/toplevel/CoreTeam.vue:107]: why this css? I hope it is unused, please remove it

**Review by aman2712 — CHANGES_REQUESTED** [review]
Please fix the small change

File: frontend/src/pages/toplevel/CoreTeam.vue (hunk 1 above)
Comment (ShantanuLamba) [reply frontend/src/pages/toplevel/CoreTeam.vue:107]: Saransh had done it before

Comment (ShantanuLamba) [reply frontend/src/pages/toplevel/CoreTeam.vue:107]: Not me

Comment (ShantanuLamba) [reply frontend/src/pages/toplevel/CoreTeam.vue:107]: The change is just the 1 line space added space </style> by me, nothing else

Comment (saransh-283) [reply frontend/src/pages/toplevel/CoreTeam.vue:107]: Its required for gap between card rows, otherwise they touch each other

Comment (ShantanuLamba) [reply frontend/src/pages/toplevel/CoreTeam.vue:107]: Wait, I will find a solution for it in bootstrap

Comment (saransh-283) [reply frontend/src/pages/toplevel/CoreTeam.vue:107]: Dont just give a margin-bottom to all membercards, it  will waste space for the lowest row

Comment (ShantanuLamba) [reply frontend/src/pages/toplevel/CoreTeam.vue:107]: I found a good solution, I am commuting it :)

Comment (ShantanuLamba) [reply frontend/src/pages/toplevel/CoreTeam.vue:107]: lets just leave it , its very less css and i wrote more bootstrap than that

**Pull request merged**